code/game/objects/items/weapons/mine.dm

```diff
@@ -61,13 +60,21 @@
 			deployed = TRUE
 			user.drop_from_inventory(src)
 			anchored = TRUE
+			armed = TRUE
 			update_icon()
-		
-	if (armed)
-		playsound(loc, 'sound/weapons/armbomb.ogg', 75, 1, -3)
-	
+
 	update_icon()
 
+/obj/item/weapon/mine/attack_hand(mob/user as mob)
+	if (deployed)
+		user.visible_message(
+				SPAN_DANGER("You have touched the mine \the [src]!"),
```

Comment:
You attempt to pick up the [src] only to hear a beep as it explodes in your hands!